src/libraries/Native/Unix/System.Globalization.Native/pal_icushim_internal.h

```diff
@@ -53,132 +53,146 @@
 #include "pal_compiler.h"
 
 #if !defined(STATIC_ICU)
+
+// #if (U_ICU_VERSION_MAJOR_NUM < 52)
+// The following APIs are not supported in the ICU versions less than 52. We need to define them manually.
+// We have to do runtime check before using the pointers to these APIs. That is why these are listed in the FOR_ALL_OPTIONAL_ICU_FUNCTIONS list.
+ U_CAPI int32_t U_EXPORT2 ucal_getWindowsTimeZoneID(const UChar* id, int32_t len,UChar* winid, int32_t winidCapacity, UErrorCode* status);
```

Comment:
Should we wrap them around a if def to check the icu header version?

Reply:
What ifdef value you are going to use? U_ICU_VERSION_MAJOR_NUM is not defined at that time to use. Having these without ifdef is ok as long as it is declared exactly same as in the ICU headers.

Reply:
Ok, sounds good.